app/models/slack_bot.rb

```diff
@@ -3,9 +3,9 @@ class SlackBot < ActiveRecord::Base
 
   def self.from_auth_hash(auth_hash)
     bot_info = auth_hash[:extra][:bot_info]
-    bot = find_by(id: bot_info[:bot_access_token])
+    bot = find_by(id: bot_info.fetch(:bot_access_token))
     if bot
-      bot.update_attributes(:access_token => bot_info[:bot_access_token])
+      bot.update_attributes(access_token: bot_info.fetch(:bot_access_token))
     else
       create(id: bot_info[:bot_access_token]) do |bot|
         bot.access_token = bot_info[:bot_access_token]
```

Comment:
Sorry I didn't explicitly called it out but let's do the same treatment everywhere we access `bot_access_token`